packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcCreate.ts

```diff
@@ -128,10 +128,10 @@ const fileNameGatherer = new SelectFileName();
 const lightningFilePathExistsChecker = new LightningFilePathExistsChecker();
 
 export async function forceLightningLwcCreate(explorerDir?: any) {
-  const outputDirGatherer = new SelectStrictDirPath(explorerDir, 'lwc');
+  const outputDirGatherer = new SelectOutputDir('lwc', true);
   const parameterGatherer = new CompositeParametersGatherer(
-    outputDirGatherer,
-    fileNameGatherer
+    fileNameGatherer,
+    outputDirGatherer
```

Comment:
Switched the order of these gatherers to keep the experience consistent with the other create commands.

Reply:
I would expect this change to break `Should create Lightning LWC` on the `scaffolding.test.ts` I fixed a few days ago.